core-plugins/widgets/GroupByAggregate-batchaggregator.json

```diff
@@ -43,7 +43,9 @@
               "Min",
               "Stddev",
               "Sum",
-              "Variance"
+              "Variance",
+              "collectList",
```

Comment:
For consistency with the other functions, let's name them CollectList and CollectSet.

Reply:
i'll make the change

Reply:
Done